pkg/types/alibabacloud/platform.go

```diff
@@ -12,6 +12,27 @@ type Platform struct {
 	// +optional
 	ResourceGroupID string `json:"resourceGroupID"`
 
+	// VpcID is the ID of an already existing VPC where the cluster should be installed.
+	// If empty, a new VPC will created for the cluster.
+	// Destroying the cluster using installer will delete this VPC.
+	// +optional
+	VpcID string `json:"vpcID,omitempty"`
+
+	// VSwitchIDs is the ID list of already existing VSwitchs where the master should be created.
```

Comment:
Globally, replace VSwitchs with VSwitches.
```suggestion
	// VSwitchIDs is the ID list of already existing VSwitches where the masters should be created.
```